### PR TITLE
Image

### PR DESCRIPTION
**Pull request by theprogrammerinyou**
Add image to component library.

### CODE REVIEW

**Review by ZuberDunge — APPROVED** [review]
Good code structure, try to use css variables for colors. Rest looks fine, good to go.

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import "../../css/variables.css";
+
+.alert {
+    margin: 1rem;
+    padding: 1rem;
+    border: 1px solid white;
+    background-color: white;
+    color: black;
+    border-radius: 10px;
```
Comment (shuntzhere) [inline components/alert/alert.css]: Replace the solid colors with variables.

**Review by shuntzhere — APPROVED** [review]
Clean code, class names are nicely named. Overall good.

**Comment by theprogrammerinyou** [comment]
Two reviews done, changed code according to suggestions, closing this PR.

**Pull request closed without merging**